Formula/luit.rb

```diff
@@ -13,11 +13,13 @@ class Luit < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}", "--with-encodings-dir=auto"
```

Comment:
Why does this change, and why is it needed?

Reply:
The `encodings-dir` is auto detected by `configure` default. If build on Linux server, the `encodings-dir` maybe `/usr/share/fonts/X11`. But this path does not exists on macOS.
It's strange behavior here, if I build on my mac, the build binary works well, but if it build from Jenkins builder, the build binary abort with message "cannot open /usr/share/fonts/X11" .